lib/errors/index.ts

```diff
@@ -16,6 +16,10 @@ const isBase = Object.fromEntries(
     Object.keys(rawErrors).map(key => [key, false])
 ) as Is;
 
+// This allows to conditionally add the old behavior of errors to properly
+//   test migration. Activate CI tests with MIGRATED=true yarn test.
+const notMigrated = (process.env.MIGRATED ?? 'false') === 'false'
```

Comment:
I feel like migrated is a little too vague. Maybe something like "ErrorIsSupported" or something.

Reply:
Updated. Please check my boolean logic, I'm easily confused on this for some reason :grimacing:

Reply:
Looks good now, much easier to make flags like this in the positive term like isX or allowY which might be why you were confused.